Content.Shared/Nutrition/Components/SharedButcherableComponent.cs

```diff
@@ -13,13 +13,23 @@ public class SharedButcherableComponent : Component, IDraggable
     {
         public override string Name => "Butcherable";
 
+        //TODO: List for sub-products like animal-hides, organs and etc?
         [ViewVariables]
-        public string? MeatPrototype => _meatPrototype;
+        [DataField("meat")]
```

Comment:
add a custom type serializer so the YAML linter can catch errors & check that the data fields correspond to actual entity IDs

```suggestion
        [DataField("meat", customTypeSerializer:typeof(PrototypeIdSerializer<EntityPrototype>))]
```